### ISSUE TITLE
Exercise 1, failed test case

### ISSUE THREAD

**Issue by ryanquek22** [nus-cs2030/2223-s1#41]
Hi there, I encountered the following issue on CodeCrunch as I was submitting Exercise 1. Does anyone know why this error is popping up? I assumed that the first line is supposed to be run from Jshell during the submission itself.

![photo_2022-09-06_17-05-44](https://user-images.githubusercontent.com/112756792/188610045-cc8b636e-9989-42dd-81cf-e2646bb46319.jpg)

**Comment by xuan-jun** [comment]
Hello, I think it could be that you have listed your `Point` class under the `main.jsh` file. 
You will need to separate your `Point` and `Circle` class into separate files namely `Point.java` and `Circle.java`. Because it seems like the expected output tried to open up the `Point.java` file but yours does not have the file.
Not sure if it is as such but if it is then I hope it helps!

**Comment by maxangyuejun** [comment]
Hi, also had the same exact problem, but I just tried resubmitting, selecting the files in different order, and that somehow solved the problem.

**Comment by danqiye1** [comment]
@ryanquek22 @maxangyuejun 

Ok this is weird. Please provide your submission ids and we can review your code.

**Comment by ryanquek22** [comment]
@danqiye1 
The submission id that yielded this error was 2354905. Thank you.

**Comment by danqiye1** [comment]
@ryanquek22 I think it is because your jsh file has a `double epsilon = 1e-15;`. It should have a static modifier.

**Comment by maxangyuejun** [comment]
@danqiye1 2338124 is the submission ID, but I've resubmitted and somehow the issue was solved automatically.

**Comment by danqiye1** [comment]
Yeah it is weird. Seems like this issue is not appearing anymore.
